.github/files/test-plugin-update/setup.sh

```diff
@@ -32,5 +32,5 @@ cp "$GITHUB_WORKSPACE/trunk/.github/files/test-plugin-update/mu-plugin.php" wp-c
 echo "::endgroup::"
 
 echo "::group::Backing up database"
-mysqldump --add-drop-database --add-drop-table --databases wordpress > "$GITHUB_WORKSPACE/db.sql"
+mysqldump --add-drop-database --add-drop-table --databases wordpress --column-statistics=0 > "$GITHUB_WORKSPACE/db.sql"
```

Comment:
Should we swap this out for `mariadb-dump`?
https://mariadb.com/kb/en/mariadb-dump/

Reply:
It looks like `mysqldump` is no longer part of MariaDB, so perhaps it's using the `mysql` version of the command now (hence the new error).

Reply:
This runs inside our dev environment docker container, which [has the `mysql-client` package installed](https://github.com/Automattic/jetpack/blob/4a55f3c47ca492fd2802113eec2acbea360153b2/tools/docker/Dockerfile#L36) for whatever reason instead of `mariadb-client`. Hence `mysqldump` exists in there while `mariadb-dump` does not.

Possibly we should change that at some point, but then we'll still have to either update a bunch of other scripts or make sure the "mysql"-named compat symlinks continue to exist.